packages/server/src/plugin/usageReporting/defaultSendOperationsAsTrace.ts

```diff
@@ -11,6 +11,7 @@ export function defaultSendOperationsAsTrace() {
   // throw away a full minute's worth of cache at once; we keep only the last
   // three minutes
   const cache = new LRUCache<string, true>({
+    max: 100,
```

Comment:
This seems small? idk.

My read of the lru-cache docs is that it's OK to specify just maxSize and not max, although you lose out on some "preallocation" advantages. I think until we have feedback from users that the preallocation matters that we should just skip to "the cache uses up to this much space" and not risk making it a lot less useful by also limiting `max`. That applies to all `max` in the PR. Thoughts?